components/match2/wikis/valorant/match_summary.lua

```diff
@@ -25,6 +27,7 @@ local NO_CHECK = '[[File:NoCheck.png|link=]]'
 
 local LINK_DATA = {
 	vlr = {icon = 'File:VLR icon.png', text = 'Matchpage and Stats on VLR'},
+	rib = {icon = 'File:RIB icon lightmode.png', text = 'Matchpage and Stats on RIB'},
```

Comment:
this change is not part of the PRs description

Reply:
Removed as other users were testing code